docs/pywbemcligeneraloptions.rst

```diff
@@ -61,166 +70,470 @@ command, general the options redefinition's are not retained between command
+      export PYWBEMCLI_SERVER=http://localhost
+      export PYWBEMCLI_NAMESPACE=root/cimv2
+      ...
+
+This ability can be used to set those environment variables and thus to persist
+the connection name in the shell environment, from where it will be used in
+any subsequent pywbemcli commands:
+
+.. code-block:: text
 
-      $ pywbemcli --mock-server classdefs.mof --mock-server insts.py --default-namespace root/myhome
-      pywbemcli> connection show
-        Name: default
-          WBEMServer uri: None
-          Default-namespace: root/myhome
-          . . .
-          use-pull-ops: either
-          pull-max-cnt: 1000
-          mock: classdefs.mof, insts.py
-          log: None
-      pywbecli> connection save --name mymockserver
-
-  See chapter :ref:`Mock WBEM server support` for more information on defining
-  mock servers.
-* **--log/-l** - See :ref:`Pywbemcli defined logging`.
-* **--verbose/-v** -  Display extra information about the processing.
-* **--version/-V** - Show the version of this command and of the pywbem package
+      $ eval $(pywbemcli -s http://localhost -u username -d root/cimv2)
+
+      $ env |grep PYWBEMCLI
+      export PYWBEMCLI_SERVER=http://localhost
+      export PYWBEMCLI_NAMESPACE=root/cimv2
+
+      $ pywbemcli server namespaces
+      . . . <list of namespaces for the defined server>
+
+.. _`--timeout general option`:
+
+--timeout general option
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The value of ``--timeout\-t`` general option is an integer that defines the
+client side timeout in seconds. The pywbem client includes a timeout mechansim
+that closes a WBEM connection if there was no activity for time period defined
+by this value. Setting the value to zero. Pywbemcli defaults to a predefined
+timeout (normally 30 seconds) if this option is not defined.
+
+
+.. _`--no-verify general option`:
+
+--no-verify general option
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the ``--no-verify/-n`` boolean general options is set, theclient does not
+verify any certificates received from the WBEM server. Any certificate returned
+by the WBEM server is accepted.
+
+
+.. _`--certfile general option`:
+
+--certfile general option
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The value of the ``--certfile`` is a file path for a X.509 client certificate
+to be presented to the WBEM server along with the ``--keyfile`` option during
+the TLS/SSL handshake. This parameter used only with HTTPS.  If None, no client
+certificate is presented to the server, enabling 1-way authentication.
+Otherwise, the client certificate is presented to the server, enabling 2-way
+(mutual) authentication. This and the ``--keyfile`` are presented to the pywbem
+:class:`pywbem.WBEMConnection` as a single parameter ``X-509``:
+
+* ``--cert_file`` - The file path of a file containing an X.509 client
+  certificate.
+* ``key_file`` - The file path of a file containing the private key belonging
+  to the public key that is part of the X.509 certificate file.
```

Comment:
We do not need to explain how this looks like at the pywbem API. This is internal implementation stuff from a perspective of the pywbemcli user.

Reply:
Gone